datatypes/Cargo.toml

```diff
@@ -11,42 +11,42 @@ repository.workspace = true
 [features]
 
 [dependencies]
-arrow = { version = "53.0", features = ["ipc_compression"] }
-arrow-array = "53.0"
-arrow-ord = "53.0"
-arrow-schema = { version = "53", features = ["serde"] }
-bytes = "1.5" # for postgres-types impls
-chrono = "0.4"
-fallible-iterator = "0.2" # only for postgres-protocol
-float-cmp = "0.9"
-gdal = "0.17"
-geo = "0.28"
-geojson = "0.24"
-image = "0.25"
-num = "0.4"
-num-traits = "0.2"
-ordered-float = { version = "4.2", features = ["serde"] }
-paste = "1.0"
-postgres-protocol = { version = "0.6" }
-postgres-types = { version = "0.2", features = [
+arrow = { workspace = true, features = ["ipc_compression"] }
```

Comment:
shouldn't we do the features globally, too?

Reply:
[43183ca](https://github.com/geo-engine/geoengine/pull/994/commits/43183ca9631ac61be216098b6affbb8d4519a1f1)

Reply:
there is still a feature defined in the macros crate https://github.com/geo-engine/geoengine/blob/fce8876e48623769438ddc4d946e6fa57ecabc9f/macros/Cargo.toml#L20

Reply:
[584de5f](https://github.com/geo-engine/geoengine/pull/994/commits/584de5f8300438ec62e235bbed5ecb1b0b8c8bdf)